docs/reference/commandline/context_inspect.md

```diff
@@ -30,27 +30,16 @@ $ docker context inspect "local+aks"
   {
     "Name": "local+aks",
     "Metadata": {
-      "Description": "Local Docker Engine + Azure AKS endpoint",
-      "StackOrchestrator": "kubernetes"
+      "Description": "Local Docker Engine",
+      "StackOrchestrator": "swarm"
```

Comment:
Need to check how this is used, but possibly we could/should reset this field and hide it